### PR TITLE
Updated /intro

### PR DESCRIPTION
**Pull request by ayiadvance**
This splits the "What are the rules for each format? What is 'Sleep Clause'?" part of /intro into two questions, both with updated links.

### CODE REVIEW

**Comment by Zarel** [comment]
This is intentional. "What is 'sleep clause'?" is an add-on question, intended to clarify what sorts of rules are being asked about. The rules _are_ explained in the tier explanation pages.

**Comment by TheJetOU** [comment]
The page is a bit outdated though since it's from XY, and I don't think there's a page like that for SM? So maybe splitting it is the only available option.

**Comment by Zarel** [comment]
I do agree with updating the link to https://www.smogon.com/dex/sm/formats/. I just don't see a reason to split it.

**Comment by TheJetOU** [comment]
Yeah, but that page doesn't really explain the clauses. Well, I guess if you click one of the formats and scroll down a bit, the clauses are explained. But the other page is way more straight forward.

**Comment by Zarel** [comment]
It literally does explain the clauses, you said so yourself.

The other page is _less_ straightforward. It doesn't explain the formats, only the clauses. The link is about the formats.

**Comment by TheJetOU** [comment]
My only gripe with your suggestion is that it says "What is Sleep Clause?" but you have to do a little bit of digging to find out what it means, but I suppose the "digging" here is minuscule, so it's not a big deal.

**Comment by HoeenCoder** [comment]
FYI:
1. This is another one of my students (so still a very to JS in general)
2. Iyarito actually asked me to make this split, I personally have no problem with the split.

File: server/chat-plugins/info.js [hunk 1]
```diff
@@ -1512,7 +1512,8 @@ const commands = {
 			`- <a href="https://www.smogon.com/forums/threads/3496279/">Beginner's Guide to Pok&eacute;mon Showdown</a><br />` +
 			`- <a href="https://www.smogon.com/dp/articles/intro_comp_pokemon">An introduction to competitive Pok&eacute;mon</a><br />` +
 			`- <a href="https://www.smogon.com/sm/articles/sm_tiers">What do 'OU', 'UU', etc mean?</a><br />` +
-			`- <a href="https://www.smogon.com/xyhub/tiers">What are the rules for each format? What is 'Sleep Clause'?</a>`
+      `- <a href="https://www.smogon.com/dex/sm/formats/">What are the rules for each format?</a><br />` +
```
Comment (HoeenCoder) [inline server/chat-plugins/info.js]: Indentation appears to be off here actually, be sure to correct that.

**Comment by Zarel** [comment]
I suppose I'm fine with the split...

**Pull request merged**